core/src/main/java/org/elasticsearch/index/query/SpanNearQueryBuilder.java

```diff
@@ -145,8 +147,8 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
 
     public static SpanNearQueryBuilder fromXContent(XContentParser parser) throws IOException {
         float boost = AbstractQueryBuilder.DEFAULT_BOOST;
-        Integer slop = null;
-        boolean inOrder = SpanNearQueryBuilder.DEFAULT_IN_ORDER;
+        int slop = DEFAULT_SLOP;
```

Comment:
given that it's an int, we wouldn't even need a specific default value but I guess you prefer it so we have an explicit constant ?